### PR TITLE
fix over-indent in Tokenizers section of docs

### LINKED ISSUE #10553: Tokenizers section over-indented on the Components page
### Rasa Open Source version

3.x

### What happened?

The "Tokenizers" section is over-indented on the Components page of the docs. Apart from the obvious inconsistency, it's causing the "Tokenizers" heading to not appear in the TOC on the right.

[Reference](https://rasa.com/docs/rasa/components#tokenizers)

### PR DESCRIPTION
**Pull request by Polaris000**
**Proposed changes**:
- fixed over-indent in the Tokenizers section of the Components page of the docs.
closes #10553

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by koernerfelicia** [comment]
Hi @Polaris000, thanks for the change, and for your patience! Could you, please, change the changelog file from `bugfix` to `doc`?  Otherwise this looks great, and we'll make sure to get it merged ASAP! 🚀

File: changelog/10553.bugfix.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixed an over-indent in the Tokenizers section of the Components page of the docs.
```
Comment (koernerfelicia) [inline changelog/10553.bugfix.md:1]: This file should be named `10553.docs.md`

**Review by koernerfelicia — CHANGES_REQUESTED** [review]

File: changelog/10553.bugfix.md (hunk 1 above)
Comment (Polaris000) [reply changelog/10553.bugfix.md]: done

**Review by koernerfelicia — APPROVED** [review]
Looks good, thank you again! 🚀

**Pull request merged**